src/types.rs

```diff
@@ -197,7 +216,68 @@ impl From<Res> for sys::Res {
 ///
 /// If an adjacent face is a pair of subfaces, the id of the first subface as encountered
 /// in a CCW traversal should be stored as the adjface id.
-pub type FaceInfo = sys::FaceInfo;
+pub struct FaceInfo(pub(crate) sys::FaceInfo);
```

Comment:
I just noticed with this change, all the `pub` members documented directly above this are now private.

Reply:
My tendency is to think that most of these `pub` files are pretty well covered by the accessor methods,
perhaps they provide simpler accessing via iterators, so perhaps we could consider adding methods like:
`fn adjacent_edge_iter()`, `fn adjacent_face_iter()`